ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -76,3 +76,81 @@ def test_binary_cross_entropy_with_logits(
         pos_weight=pos_weight[0],
         reduction=reduction,
     )
+
+#mse
+@handle_test(
+    fn_tree="mse_loss",
```

Comment:
You should specify the full function tree `functional.ivy.experimental` is missing

Reply:
Thanks for pointing out the glitch @hirwa-nshuti !

I've updated the fn_tree with full function tree i.e functional.ivy.experimental.mae_loss and functional.ivy.experimental.mse_loss